data_scraper/tiingo.py

```diff
@@ -125,7 +125,12 @@ def _merge(symbol, symbol_df):
                          parse_dates=["date"],
                          index_col="date")
     symbol_df.index = symbol_df["date"]
-
+    sequence = [
```

Comment:
This logic should be moved up to where we save each scraped symbol. 
Consider changing the `expected_columns` to our preferred format (the one you are using in `sequence`), and adding the reindex operation before saving the csv.